libr/core/cmd.c

```diff
@@ -4631,7 +4633,13 @@ static char *do_handle_ts_unescape_arg(struct tsr2cmd_state *state, TSNode arg)
 		return do_handle_ts_unescape_arg (state, ts_node_named_child (arg, 0));
 	} else if (is_ts_arg_identifier (arg)) {
 		return unescape_arg (state, arg, SPECIAL_CHARS_REGULAR);
-	} else if (is_ts_quoted_arg (arg)) {
+	} else if (is_ts_single_quoted_arg (arg)) {
+		char *c = unescape_arg (state, arg, SPECIAL_CHARS_SINGLE_QUOTED);
+		c[strlen (c) - 1] = '\0';
```

Comment:
If len ==0 oobread

Reply:
Err oobwrite

Reply:
yeah let me re-write this part. len == 0 can't actually happen though because being a quoted argument, `c` contains at least `''`.